infra/modules/providers/azure/app-monitoring/variables.tf

```diff
@@ -63,7 +63,7 @@ variable "metric_alert_criteria_threshold" {
   type        = string
 }
 
-variable "scaling_values" {
+variable "monitoring_dimension_values" {
   description = "App instance names made available from app service plan scaling options."
```

Comment:
Nit: I don't know what this is from the description given. Can we explain this better?

Reply:
Clarified to `Dimensions used to determine service plan scaling.`